ui/raidboss/popup-text.ts

```diff
@@ -122,19 +145,22 @@ const triggerUpperCase = (str: string): string => {
 const onTriggerException = (trigger: ProcessedTrigger, e: any) => {
   // When a fight ends and there are open promises, from delaySeconds or promise itself,
   // all promises will be rejected.  In this case there is no error; simply return without logging.
-  if (!e)
+  if (!e) {
```

Comment:
I am a bit surprised that eslint doesn't fix up the curly braces.  It usually autofixes these.  Is this the curly problem you're mentioning?

Reply:
prettier will put this one-line code block in the same line with if statement.

Reply:
So, if I turn this code into `if (!e) return` then this is a https://eslint.org/docs/rules/nonblock-statement-body-position error, and eslint will put them on separate lines.  Are there cases where this rule doesn't work?

Reply:
If I change formatter to be:
```js
const linter = new eslint.ESLint({
  fix: true,
  overrideConfig: {
    rules: {
      'max-len': 'off',
    },
  },
});
```

...and then `git checkout main .eslintrc.cjs` and then `node util/formatter.cjs`, then `git diff ui/raidboss/popup-text.ts` looks better to me.  What do you think?

Reply:
should work fine now?